src/itemrenderer.cpp

```diff
@@ -47,9 +47,11 @@ void prepare_header(
 	};
 
 	const std::string feedtitle = item_renderer::get_feedtitle(item);
-	add_line(feedtitle, _("Feed: "));
-	add_line(utils::utf8_to_locale(item->title()), _("Title: "));
-	add_line(utils::utf8_to_locale(item->author()), _("Author: "));
+	add_line(utils::quote_for_stfl(feedtitle), _("Feed: "));
```

Comment:
We already have test coverage for `ItemRenderer`. Can you please add a regression test for these changes?

Reply:
Done